### PR TITLE
Use latest ami id for arm_suse15 in suse OS family

### PR DESCRIPTION
**Pull request by vasireddy99**
**Description:** 

This PR is to use latest arm_suse15 available for the tests, using latest ami availble in community closest to the pattern since `suse-sles-15-sp2-v20200721-hvm-ssd-arm64*` doesn't any updated matching ami's available.

image

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

**Review by bryan-aguilar — CHANGES_REQUESTED** [review]
Is the search pattern used in this branch? I thought we had changed to using a fixed `ami_id` and did not revert back to a search pattern until post `v0.22.x`. I believe this PR as is would cause an error because we use the fixed [`ami_id`](https://github.com/aws-observability/aws-otel-test-framework/blob/6ef782319001f5ac82cb760f820a530db60c7acf/terraform/ec2/main.tf#L67) rather than the search pattern. See [this](https://github.com/aws-observability/aws-otel-test-framework/commit/e6a61633894e9706bb621498d3e74acf6f5767a3#diff-74d4b9e2f0f97a5797874df730dacab3d04096afa0b81a7cccdc630802dde5d8) commit for when that change was made for additional context.

File: terraform/ec2/amis.tf [hunk 1]
```diff
@@ -300,13 +300,13 @@ EOF
...
-      ami_owner          = "amazon"
+      ami_search_pattern = "suse-sles-15-sp2-v20210212-hvm-ssd-arm64*"
+      ami_owner          = "013907871322"
+      ami_id             = "ami-0b704e615e7364682"
       ami_product_code   = []
       family             = "linux"
-      login_user         = "ec2-user"
       arch               = "arm64"
-      instance_type      = "t4g.micro"
+      instance_type      = "t4g.nano"
```
Comment (bryan-aguilar) [inline terraform/ec2/amis.tf:309]: Why was this change made?

File: terraform/ec2/amis.tf [hunk 2]
```diff
@@ -300,13 +300,13 @@ EOF
 
     arm_suse15 = {
       os_family          = "suse"
-      ami_search_pattern = "suse-sles-15*"
-      ami_owner          = "amazon"
+      ami_search_pattern = "suse-sles-15-sp2-v20210212-hvm-ssd-arm64*"
+      ami_owner          = "013907871322"
```
Comment (bryan-aguilar) [inline terraform/ec2/amis.tf:304]: What owner does this reference?

Comment (bryan-aguilar) [reply terraform/ec2/amis.tf:304]: Saw the updated PR description.

File: terraform/ec2/amis.tf (hunk 1 above)
Comment (vasireddy99) [reply terraform/ec2/amis.tf:309]: reverted this change

**Review by humivo — APPROVED** [review]

**Pull request merged**